format.js

```diff
@@ -307,6 +389,7 @@ function makeEncryptionFormat() {
     if ((plaintextBuf = unboxWith(groupKeys, ATTEMPT1))) return plaintextBuf
     if ((plaintextBuf = unboxWith(selfKey, ATTEMPT16))) return plaintextBuf
     if ((plaintextBuf = unboxWith(dmKey, ATTEMPT16))) return plaintextBuf
+    if ((plaintextBuf = unboxWith(poBoxKeys, ATTEMPT16))) return plaintextBuf
```

Comment:
do we need to check all 16? not sure where the pobox stuff is specified, but i could maybe find that

Reply:
it's a bit vague but it does sort of sound like any slot is allowed https://github.com/ssbc/private-group-spec/tree/master/po-box

> When I see a pobox_id listed in content.recps, I derive a key for the envelope key_slot like so:

Reply:
and in ssb-tribes we did check all 16 https://github.com/ssbc/ssb-tribes/blob/134064ba84e7d49c292349aa00aeccd42f2103b4/envelope.js#L126

Reply:
done